shared/html/devtools-panel.html

```diff
@@ -0,0 +1,176 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <title>DuckDuckGo Privacy Essentials Debugger</title>
+  <style type="text/css">
+    @font-face {
+      font-family: 'DDG_ProximaNova';
+      src: url('../public/font/ProximaNova-Reg-webfont.woff') format('woff');
+      font-weight: normal;
+      font-style: normal;
+    }
+
+    @font-face {
+      font-family: 'DDG_ProximaNova';
+      src: url('../public/font/ProximaNova-Sbold-webfont.woff') format('woff');
+      font-weight: 600;
+      font-style: bold;
+    }
+
+    body {
+      font-family: 'DDG_ProximaNova';
+      color: #000
+    }
+
+    .header {
+      padding: 10px;
+    }
+
+    .block {
+      background-color: #DE5833;
+    }
+
+    .redirect {
+      background-color: #FDD20A;
+    }
+
+    .ignore {
+      background-color: #CACACA;
+    }
+
+    .cookie {
+      background-color: #9CB8FF
+    }
+
+    .set-cookie {
+      background-color: #BDA8FC
+    }
+
+
+    table {
+      table-layout: fixed;
+      width: 100%;
+      border-collapse: collapse;
+      border: 1px solid black;
+    }
+
+    thead th:nth-child(1) {
+      width: 2%;
+    }
+
+    thead th:nth-child(2) {
+      width: 48%;
+    }
+
+    thead th:nth-child(3) {
+      width: 18%;
+    }
+
+    thead th:nth-child(4) {
+      width: 22%;
+    }
+
+    thead th:nth-child(5) {
+      width: 10%;
+    }
+
+    th,
+    td {
+      padding: 5px;
+      white-space: nowrap;
+      text-overflow: ellipsis;
+      overflow: hidden;
+    }
+
+    tr > td:nth-child(4) {
+      overflow: visible;
```

Comment:
Allowing this to wrap works better on smaller screens without colliding with the next column. For the columns that use ellipsis, you could use title so it's still readable on hover perhaps.

```suggestion
word-wrap: anywhere;
overflow: auto;
white-space: break-spaces;
```